lardataobj/RecoBase/OpHit.cxx

```diff
@@ -17,6 +17,8 @@ namespace recob{
     , fFrame         (0 )
     , fPeakTime      (0 )
     , fPeakTimeAbs   (0 )
+    , fStartTime     (0 )
+    , fRiseTime      (0 )
```

Comment:
This means that a default-constructed `OpHit` object will return `HasStartTime()` of `true`, which is probably not what you want.  Instead you should do:

```suggestion
    , fStartTime     (DefaultTime)
    , fRiseTime      (DefaultTime)
```
which will also catch the cases where `OpHit`s are read from old files.